package/gql/codegen.ts

```diff
@@ -0,0 +1,18 @@
+import type { CodegenConfig } from "@graphql-codegen/cli";
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: "https://optimism-sepolia.easscan.org/graphql",
+  documents: "src/**/*.(ts|tsx)",
+  generates: {
+    "src/eas/generated.ts": {
+      plugins: [
+        "typescript",
+        "typescript-operations",
+        "typescript-graphql-request",
+      ],
+    },
+  },
+};
+
```

Comment:
❌ think this is not a good choice anymore, I suggest using https://github.com/0no-co/gql.tada like https://github.com/bleu/cow-stop-loss/blob/main/src/lib/ponderApi/queries.ts did

Reply:
we just solved this together https://github.com/bleu/op-bacchus/pull/8/commits/c90793dc1e27d8d348ec69858ca17e21dcace34e